start.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Set the environment variables.
+source ./set_env_variables.sh
+
+build=false
+
+# If the `-b` flag is passed, set build to true.
+while getopts b: flag; do
+    case ${flag} in
+        b) build=true;;
+    esac
+done
+
+check_status() {
+    status_code=$(curl --write-out %{http_code} --silent --output /dev/null localhost:${FLASK_RUN_PORT}/graphiql)
```

Comment:
This line should check if the rStudio page is available. I believe it is at localhost:8787.
Please ensure there is an accurate environment variable related to the port and that it isn't trying to access graphiql.